contracts/multisig_wallet.scilla

```diff
@@ -77,19 +77,16 @@ type ProxyTransaction =
 | Pause
 (* unpause () *)
 | Unpause
+(* changeProxyAdmin(newAdmin : ByStr20) *)
 | ChangeProxyAdmin of ByStr20
 (* update_admin(admin : ByStr20)  *)
 | UpdateAdmin of ByStr20
 (* update_verifier (verif : ByStr20)  *)
 | UndateVerifier of ByStr20
 (* drain_contract_balance ()  *)
 | DrainContractBalance
-(* update_minstake (min_stake : Uint128) *)
-| UpdateMinStake of Uint128
-(* update_maxstake (max_stake : Uint128) *)
-| UpdateMaxStake of Uint128
-(* update_contractmaxstake (max_stake : Uint128) *)
-| UpdateContractMaxStake of Uint128
+(* update_staking_parameter (minx_stake : Uint128, max_stake : Uint128, contract_max_stake : Uint128) *)
```

Comment:
I think there is a typo . `minx_stake` should be `min_stake`

Reply:
Ok, let me fix it.